test/ElasticSearchIndexerTest.php

```diff
@@ -88,6 +89,49 @@ public function testGetIndexNameByDocumentForUnclassifieds(): void
         $this->assertSame(IndexerInterface::UNCLASSIFIEDS_DOCUMENTS_INDEX, $indexName);
     }
 
+    public function testBuildIndexBulkErrorResponse(): void
```

Comment:
It is missing cover the second exception.

Reply:
Done